src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/barcode/BarcodeRecipientRepository.kt

```diff
@@ -0,0 +1,45 @@
+package uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.barcode
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import java.time.LocalDate
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.OneToOne
+import javax.persistence.Table
+import javax.validation.constraints.NotNull
+
+@Repository
+interface BarcodeRecipientRepository : JpaRepository<BarcodeRecipient, Long> {
+  fun getByBarcode(barcode: Barcode): BarcodeRecipient?
+}
+
+@Entity
+@Table(name = "barcode_recipients")
+data class BarcodeRecipient(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @NotNull
```

Comment:
I'd still like to remove all of these at some point - code that does nothing just confuses people form the future,